config/jobs/periodic/kubeflow/kubeflow-pipelines-component.yaml

```diff
@@ -0,0 +1,23 @@
+  periodics:
+  - name: kubeflow-pipelines-component-yaml-ppc64le
+    cluster: kubeflow-ppc64le-cluster
+    labels:
+      preset-kubeflow-powervs: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
```

Comment:
Hope it is an IBM COS S3 bucket, If so, the `bucket` value was supposed to start withs3://
Also I think it has to be `s3_credentials_secret` instead of `gcs_credentials_secret` like in https://github.com/ppc64le-cloud/test-infra/blob/master/config/prow/config.yaml#L78-L80

Reply:
Did you try pushing the logs to that bucket by disabling the `--local` flag at https://github.com/ppc64le-cloud/test-infra/blob/master/hack/test-pj.sh#L16 and stop pushing artifacts to `out_dir`

Reply:
If the logs are pushed properly to your bucket, the test-pj runs should appear in https://prow.ppc64le-cloud.cis.ibm.net/job-history/s3/kubeflow-ppc64le-s3-credentials/logs/kubeflow-pipelines-component-yaml-ppc64le

Also, we would need your s3-credentials on our prow cluster for UI components like deck to interact and display UI.

Meanwhile, I need to check on how to add multiple s3 buckets to these prow components.

Reply:
I have tried pushing the locally , I'm trying it now . I'll share the s3 credentials locally with u

Reply:
hey @valen-mascarenhas14 it seems we can use our existing s3 bucket for this job too.
So, you may remove this gcs_configuration altogether.
Apologies for the extra effort.